core/templates/dev/head/pages/exploration_editor/statistics_tab/early_quit_issue_directive.html

```diff
@@ -0,0 +1,34 @@
+<div class="row oppia-issues-row-eq-height oppia-issues-bor-left oppia-issues-issue-dir">
+  <div class="col-md-3 oppia-issues-bor-btm oppia-issues-bor-right text-center oppia-pt2 oppia-pb2 oppia-issues-bg-clr oppia-issues-text-white">
+    <div style="font-size: 1.3em;">
+      Issue <[currentIssueIndex + 1]>
```

Comment:
Let's just introduce a `$scope.currentIssueIdentifier` variable instead of doing this math.

Reply:
Done.